src/TLuaInterpreterMapper.cpp

```diff
@@ -3528,6 +3528,42 @@ int TLuaInterpreter::setExitStub(lua_State* L)
     host.mpMap->update();
     return 0;
 }
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#setExitWeightFilter
+// Scripts may return false or the string "block" from the callback.
+// Returning those values prevents the exit from being added to the pathfinding graph.
```

Comment:
Can it adjust the exit weight at all or does it only block an exit from being used?